docs/reference/components.cue

```diff
@@ -300,25 +300,13 @@ components: {
 		}
 
 		how_it_works: {
-			environment_variables: {
```

Comment:
Did this go somewhere else?

Reply:
This will become a default section. I'm contemplating removing this as it's confusing. For the reference docs, having a section called "Environment variables" is misleading since users will probably expect it to contain a list of supported environment variables.